src/libraries/Native/Unix/configure.cmake

```diff
@@ -447,10 +447,11 @@ check_symbol_exists(
     HAVE_DISCONNECTX)
 
 set(PREVIOUS_CMAKE_REQUIRED_FLAGS ${CMAKE_REQUIRED_FLAGS})
-set(CMAKE_REQUIRED_FLAGS "-Werror -Wsign-conversion")
+set(CMAKE_REQUIRED_FLAGS "-Werror=sign-conversion")
```

Comment:
Can you please explain this change? We want error on all warnings.

Reply:
It is related to other warnings caused by uClibc headers with new compilers, so the change is needed to catch only the warning relevant for the check

Reply:
We want to error on all warnings that are enabled. Our policy w.r.t. warning has been to fix them if reasonably feasible or disable the specific ones where fixing is not possible or too cumbersome. It would be good to get a complete list of warnings that we are getting to reason about what to do here. It seems that the the warnings are uClibs rather than armel specific. So we could start with disabling them all just when compiling against uClibc. Is there a reasonable way to detect we are compiling against uCLibs at build time?

Reply:
I have tried to remove this change too and the build succeeded without any warnings or errors.

Reply:
It might depend on the gcc version used, similarly to `-Wno-atomic-alignment`
I can revert this. However, i believe that the check should only care about `sign-conversion` related errors since it's checking whether or not the flags are signed